### ISSUE TITLE
Is Vuejs new version backword compatible ?

### ISSUE THREAD

**Issue by Anshul-Batra** [vuejs/Discussion#409]
@yyx990803  

Hi Evan,

I am currently developing a project using Vuejs but there are few this for which i am looking for solution ans i found those fixes in new version of vuejs. Please let me know if i update my Vuejs file to latest versionm will my project work fine.

is Vuejs backword compatible?

The version that i am using currently is:
Vue.js v0.11.4

Thanks,
Anshul

**Comment by Mat-Moo** [comment]
You should be ok going to 0.12 but not 1 due to syntax changes and other changes

**Comment by yyx990803** [comment]
No, 0.12 is not compatible with 0.11. You will need to checkout the changelog for the breaking changes: https://github.com/yyx990803/vue/releases/tag/0.12.0 (and 0.12.1 / 0.12.2 also has a few breaking changes, but versions after that are mostly compatible)

**Comment by mark-hahn** [comment]
You should consider going to semantic versioning.

**Comment by simplesmiler** [comment]
@mark-hahn Vue does follow semantic versioning.
> Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.

**Comment by mark-hahn** [comment]
It would be nice to change y when anything breaks, even if the rules don't
require it.

**Comment by yyx990803** [comment]
@mark-hahn well post 1.0 there will be no more confusions. If something breaks, it's 2.0!

**Comment by mark-hahn** [comment]
> If something breaks, it's 2.0!

I hope you are joking.  :-)    It would be 1.1.0.

**Comment by yyx990803** [comment]
In fact, semver demands major version bump every time you make a breaking
change.

**Comment by mark-hahn** [comment]
Thanks.  I was mistaken.  Sorry.
